src/__test__/addcomment.test.js

```diff
@@ -0,0 +1,10 @@
+import addComment from '../module/addcomment.js';
+
+jest.mock('../src/module/addcomment.js');
+
+describe('Test Add Comment Function', () => {
+  it('Add Comment ', async () => {
+    const status = await addComment(1);
+    expect(status).toBe('created');
+  });
+});
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly make sure all your tests are passing. This will make your project solid and fully tested.
![tests](https://user-images.githubusercontent.com/63917725/189488817-d4a901da-c5e1-4907-a0db-04607031c36f.PNG)